### ISSUE TITLE
[Tarea 1] Política de atrasos

### ISSUE THREAD

**Issue by fernandamansilla** [UC-IIC3253/2021#30]
Hola , quería saber si existe alguna política de descuento por atraso para la Tarea 1?

**Comment by mugartec** [comment]
Hola @fernandamansilla. No, no hay una política de descuento por atraso.

Saludos.

**Comment by fernandamansilla** [comment]
Gracias por responder. Para estar segura, entonces eso significa que si la tarea se entrega después de las 23:59 de hoy es inmediatamente un 1.0 ?

**Comment by marceloarenassaavedra** [comment]
Sí, el plazo para la entrega de la tarea es hoy a las 23:59 hrs.

Saludos!
